well-formed-strings.rb

```diff
@@ -0,0 +1,23 @@
+require 'pry'
+class StringValidator
+  def validate(input)
+    result = false
+    match = {"}" => "{", "]" => "[", ")" => "("}
```

Comment:
Can this be pulled out in any manner? Can it change? Does it change? Is it reusable?